draft-aegis-mimi-arch-latest.md

```diff
@@ -311,13 +379,13 @@ HTTP PATCH /apps/{appId}/clients/{clientId}
 Input:
 
 {
-   clientIdentity: Blob,
+   ClientID: Blob,
    tagsToAdd: Tags,
    tagsToRemove: Tags
 }
 ~~~
 
-Patching an existing client MUST fail if changing `clientIdentity` results in a
+Patching an existing client MUST fail if changing `ClientID` results in a
```

Comment:
As above, `clientIdentity` was correct. `clientId` doesn't clash by default because it can't change.